vendor/vendor.json

```diff
@@ -92,18 +92,18 @@
 			"revision": "89a49455acb77a90f228cdda701be3d8cc9083d4",
 			"revisionTime": "2018-08-24T08:50:24Z"
 		},
+		{
+			"checksumSHA1": "qRvSbaSsoiLcQ4PRrm3R5XcAoUs=",
+			"path": "github.com/aliyun/alibaba-cloud-sdk-go/services/drds",
+			"revision": "aa91a73c945754f8d139600027dbaa4eb64ecb61",
+			"revisionTime": "2018-10-10T04:10:18Z"
+		},
 		{
 			"checksumSHA1": "iR51yUh0y0IsOoEtICvSsGf/MEM=",
 			"path": "github.com/aliyun/alibaba-cloud-sdk-go/services/ecs",
 			"revision": "6b2114e129d0d7a672cd434b386b5e54af41d9b9",
 			"revisionTime": "2018-08-15T11:45:37Z"
 		},
-		{
-			"checksumSHA1": "HZzI30CNPIUOv+Ss9/oUoHMssqE=",
-			"path": "github.com/aliyun/alibaba-cloud-sdk-go/services/ess",
-			"revision": "c6478429a81ca006934d787d9770a522da4ae745",
-			"revisionTime": "2018-09-06T08:00:34Z"
-		},
 		{
```

Comment:
Why the PR removes the ess sdk?

Reply:
i don't now , why ess sdk remove ,but  now ,i has add this to vendor.json